src/tcgwars/logic/impl/gen8/DarknessAblaze.groovy

```diff
@@ -1453,6 +1454,7 @@ public enum DarknessAblaze implements LogicCardInfo {
           attackRequirement {}
           onAttack {
             damage 30
+            switchYourActive may:true
```

Comment:
Does this need parenthesis?

Reply:
Since I am providing arguments and it isn't nested in another method, I _should not_ need parentheses. If it causes issues it'll be a quick fix at least.